public/app/frontend/src/components/button/button.scss

```diff
@@ -0,0 +1,10 @@
+@import '../../../style';
```

Comment:
@EarthlingDavey possibly. I've actually been looking at ways for us to avoid having to import everything repeatedly as Sass is deprecating @import in favour of @use https://sass-lang.com/documentation/at-rules/import/

I'm going to leave this as is for now but will make an improvement ticket.